funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -276,8 +276,7 @@ def _task_puller_loop(self, quiesce_event):
             try:
                 if int(time.time() - self.last_heartbeat) > self.heartbeat_threshold:
                     log.critical(
-                        "[TASK_PULL_THREAD] Missed too many heartbeats. "
-                        "Setting quiesce event."
+                        "Missed too many heartbeats. " "Setting quiesce event."
                     )
```

Comment:
No need for the compiler string concatenation?

Reply:
This is probably just a `black` artifact. It happens pretty often when strings change length and then `black` doesn't want to make any changes to the AST (even though it would help).

Try a file with
```python
s = (
    "some                   very                        long           str "
    "other                    long                        str"
)
```
and convert to
```
s = (
    "some str "
    "other str"
)
```
and run `black` on it. You'll see a rewrite just like this.

There's no great solution as far as I know. Like I mentioned, `black` insists on not changing AST for safety, and finding these with some post-fixing libcst thing has just not been a priority for anyone as far as I know.